packages/nimble-components/CONTRIBUTING.md

```diff
@@ -18,13 +18,11 @@ The library is built on the open source [FAST Design System library](https://fas
 
 ## Getting started
 
-From the `nimble` directory:
```

Comment:
Looks like we've duplicated this info in the top-level `CONTRIBUTING.md` as well. Probably easiest to keep mirroring the info…

Reply:
+1 I'd advocate to have README tweaks in a follow-on PR to not have to cycle language revisions here